ads/google/a4a/performance.js

```diff
@@ -194,36 +224,63 @@ export class GoogleAdLifecycleReporter extends BaseLifecycleReporter {
    * @override
    */
   sendPing(name) {
-    this.emitPing_(this.buildPingAddress_(name));
+    this.emitPing_(this.buildPingAddress_(name, this.extraVariables_));
```

Comment:
Given emitPing is only ever referenced within this function, why not just move the contents of emitPing to be within sendPing?